res/css/photon/checkbox.css

```diff
@@ -0,0 +1,57 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+.photon-checkbox {
+  position: relative;
+  padding: 0;
+  border: 1px solid var(--grey-90-a30);
+  -webkit-appearance: none;
+  -moz-appearance: none;
+  appearance: none;
+  background-color: var(--grey-90-a10);
+  border-radius: 2px;
+}
+
+/* Use this class with the photon-checkbox for the canonical photon styling. */
+.photon-checkbox-default {
+  width: 16px;
+  height: 16px;
+  margin: 0 4px;
+}
+
+/* Use this class with the photon-checkbox for the custom micro variant. */
+.photon-checkbox-micro {
+  width: 13px;
+  height: 13px;
+  margin: 0 3px;
```

Comment:
I see this is overriden by `margin: 0 5px` in `.stackSettingsCheckbox`; I think we should remove it in `.stackSettingsCheckbox` and use this value instead :)